cypress/integration/sortTableDom.spec.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+Cypress.Commands.add('isSorted', (columnName, clolumnNumber) => {
+  cy.contains(columnName).click();
+
+  cy.get(`tr:nth-child(n) td:nth-child(${clolumnNumber})`).then(($fcolumn) => {
+    const columns = [...$fcolumn].map((column) =>
+      column.innerText.replace('$', '').replace(',', ''));
+    let counter = 0;
+
+    for (let i = 0; i < columns.length; i++) {
+      if (isNaN(Number(columns[i]))) {
+        if (columns[i + 1] >= columns[i]) {
+          counter += 1;
+        }
```

Comment:
columns[i] is a string? right? why don't we use localeCompare?

Reply:
done